homeassistant/components/insteon/insteon_entity.py

```diff
@@ -75,7 +76,14 @@ def name(self):
     @property
     def extra_state_attributes(self):
         """Provide attributes for display on device card."""
-        return {"insteon_address": self.address, "insteon_group": self.group}
+        ramprate = self.get_device_property(RAMP_RATE)
+        onamount = self.get_device_property(ON_LEVEL)
```

Comment:
I'm not an python expert by any means but I believe these should be named like_this. So, on_amount and ramp_rate